cypress/integration/ampSpec.js

```diff
@@ -42,6 +42,14 @@ describe('AMP Tests', () => {
     ampScript.should('have.attr', 'src', 'https://cdn.ampproject.org/v0.js');
   });
 
+  it('should not have any non-amp scripts in the body or the head', () => {
+    //  The reason we expect '2' is because Cypress injects a script into the head, which we have had to allocate for.
```

Comment:
Looking at it again I'd move this line down to line 50 (otherwise it suggests we expect 2 scripts in the `body script`)

So:

```js
.should('be', 2); // 1 for amp.js + 1 that Cypress injects into the head
```